src/tbots.py

```diff
@@ -0,0 +1,86 @@
+#!/opt/tbotspython/bin/python3.8
+
+import os
+import sys
+import iterfzf
+from subprocess import PIPE, run, check_call
+import argparse
+from thefuzz import fuzz
+from thefuzz import process
+
+THEFUZZ_MATCH_RATIO_THRESHOLD = 50
```

Comment:
nit: I'm not very familiar with `fuzz`, could you add a comment explaining what this represents?

Reply:
added a comment